code/datums/outfits/outfit_admin.dm

```diff
@@ -1385,3 +1385,42 @@
 	mask = /obj/item/clothing/mask/breath/vox
 	belt = /obj/item/tank/internals/emergency_oxygen/double/vox
 	box = /obj/item/storage/box/survival_vox
+
+/datum/outfit/admin/enforcer
+	name = "Oblivion Enforcer"
+
+	uniform = /obj/item/clothing/under/color/white
+	shoes = /obj/item/clothing/shoes/white/enforcer
+	back = /obj/item/storage/backpack/satchel
+	id = /obj/item/card/id
```

Comment:
I would suggest giving them the "/obj/item/card/id/data" since it doesnt have the NT logo in it.